### PR TITLE
P1518R2 Stop overconstraining allocators in container deduction guides

### LINKED ISSUE #4645: [2021-06 LWG Motion 8] P1518R2 Stop overconstraining allocators in container deduction guides
[P1518R2](https://wg21.link/p1518r2)

### PR DESCRIPTION
**Pull request by jensmaurer**
Fixes #4645
Fixes cplusplus/papers#381

### CODE REVIEW

**Review by JohelEGP — CHANGES_REQUESTED** [review]
Between `vector<bool>` and `multimap`, you missed the `map` changes.

**Comment by jensmaurer** [comment]
@JohelEGP , thanks, fixed.

**Review by jwakely — APPROVED** [review]

File: source/containers.tex [hunk 1]
```diff
@@ -9729,7 +9729,7 @@
 \item It has an \tcode{InputIterator} template parameter and a type that does not qualify as an input iterator is deduced for that parameter.
 \item It has a \tcode{Compare} template parameter and a type that qualifies as an allocator is deduced for that parameter.
 \item It has a \tcode{Container} template parameter and a type that qualifies as an allocator is deduced for that parameter.
-\item It has an \tcode{Allocator} template parameter and a type that does not qualify as an allocator is deduced for that parameter.
```
Comment (tkoeppe) [inline source/containers.tex:9732]: This deletion is at odds with the paper, and the paper is at odds with the status quo. @Quuxplusone, could you perhaps check if the paper is misrepresenting the status quo and was missing this item? (And the item is meant to be retained?)

Comment (tkoeppe) [reply source/containers.tex:9732]: Never mind, I misread the diff. All good!

**Pull request merged**